### PR TITLE
docs: sla

### PR DESCRIPTION
**Pull request by vasco-santos**
Adds SLA

### CODE REVIEW

File: src/pages/docs/service-level-agreement.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## 1. web3.storage Service Level Agreement ("SLA").
+
+Nitro Data LLC ("Company") commits to provide a level of service for web3.storage Customers demonstrating:
+
+1.1 99.9% Uptime. The Upload Service will upload Customer Content at least 99.9% of the time and the Read Service will serve Customer Content at least 99.9% of the time, subject to Section 4.
+
+1.2 Penalties. If any of the Services fails to meet the above service level, the Customer will receive a Service Discount on their next monthly bill, as further described in Section 6 of this SLA.
+
+1.3 Upload Size Limit. Individual uploads to the Upload Service may have a size limit per API call, as described in the API documentation available at <https://web3.storage/docs/reference/http-api/>.
```
Comment (olizilla) [inline src/pages/docs/service-level-agreement.md:11]: We don't have an http API reference

File: src/pages/docs/service-level-agreement.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+1.4 Tier-Based Size Limit. Individual uploads are subject to size limits based on Customer’s chosen pricing tier as described in the Pricing page available at <https://web3.storage/account/payment/>.
```
Comment (olizilla) [inline src/pages/docs/service-level-agreement.md:13]: Link to pricing pricing page?

File: src/pages/docs/service-level-agreement.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+2.6 "Incident" means any set of circumstances resulting in a failure to meet a Service Level.
+
+2.7 "Outage Period" is equal to the number of downtime minutes resulting from an Unscheduled Service Outage.
+
+2.8 “Peered” means a persisted connection between peers configured on IPFS.
+
+2.9 "Services" means, collectively, the Upload Service and the Read Service.
+
+2.10 “Upload Service” refers to uploads through the HTTP API for file types and sizes supported by the API, as defined by the API docs available at <https://web3.storage/docs/reference/http-api/>.
```
Comment (olizilla) [inline src/pages/docs/service-level-agreement.md:35]: No link available

File: src/pages/docs/service-level-agreement.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+(g) Caused by Customer attempting to upload incompatible data types or raw files larger than 100 MB via the Upload Service;
+
+(i) Caused by Customer attempting to upload files via the web3.storage website;
+
+(j) Caused by Customer attempting to store files via the pinning service API;
+
+(k) Due to failures of infrastructure used to read data from web3.storage, including inaccessibility of public gateways;
+
+(l)  That resulted from Customer’s use of gateways not peered with web3.storage (i.e., not listed on <https://github.com/web3-storage/web3.storage/blob/main/PEERS>);
```
Comment (olizilla) [inline src/pages/docs/service-level-agreement.md:81]: What to do with the peers file? Migrate to w3up?

**Review by olizilla — CHANGES_REQUESTED** [review]

File: src/pages/docs/service-level-agreement.md (hunk 2 above)
Comment (vasco-santos) [inline src/pages/docs/service-level-agreement.md:13]:
```suggestion
1.4 Tier-Based Size Limit. Individual uploads are subject to size limits based on Customer’s chosen pricing tier as described in the Pricing page available at <https://web3.storage/pricing/>.
```

File: src/pages/docs/service-level-agreement.md (hunk 1 above)
Comment (vasco-santos) [reply src/pages/docs/service-level-agreement.md:11]: @heyjay44 will see we should drop this point

File: src/pages/docs/service-level-agreement.md (hunk 3 above)
Comment (vasco-santos) [reply src/pages/docs/service-level-agreement.md:35]: @heyjay44 will see we should drop this point or rephrase to w3up

File: src/pages/docs/service-level-agreement.md (hunk 4 above)
Comment (vasco-santos) [reply src/pages/docs/service-level-agreement.md:81]: will talk with alan

File: src/pages/docs/service-level-agreement.md (hunk 1 above)
Comment (heyjay44) [reply src/pages/docs/service-level-agreement.md:11]: 1.3 Upload Size Limit. Individual uploads to the Upload Service may encounter issues if they are over 700GB.

File: src/pages/docs/service-level-agreement.md (hunk 3 above)
Comment (heyjay44) [reply src/pages/docs/service-level-agreement.md:35]: 2.10 “Upload Service” refers to uploads through the website, client, and CLI for supported file types and sizes as defined by the docs available at [https://web3.storage/docs/](https://web3.storage/docs/reference/http-api/).

File: src/pages/docs/service-level-agreement.md (hunk 1 above)
Comment (vasco-santos) [reply src/pages/docs/service-level-agreement.md]: we changed to 750

**Review by heyjay44 — APPROVED** [review]

**Pull request merged**